README.md

```diff
@@ -50,7 +50,11 @@ You'll need the following set up on your system:
     ```
 
     ```
-    npm install supertest
+    npm install supertest --save-dev
+    ```
+
+    ```
+    npm install jest-sorted --save-dev
```

Comment:
If you already have them listed as dependencies or dev dependencies in your package.json then all someone should need to do is "npm i"